README.md

```diff
@@ -7,7 +7,7 @@
 When developers [Kristi](https://github.com/Kristiannmiller), [Brigette](https://github.com/BrigetteDoelp), and [Nathaniel](https://github.com/nathanielmillard), were thinking of an app they would want to build together, Rume was born of an interesting in creating a design specific app that helped an already emotionally conscious user do more than "get past" or deescalate feelings, and rather create a "space" that validates a user where they are at. We at Rume, believe that it's just important to take time and feel what you are feeling, as it is to know how to guide yourself back to peace and calm.
```

Comment:
We should take a look through the readme on one of these refactor branches and fix up the wording a little. I see a few grammatical errors that can be easily fixed.